src/File.php

```diff
@@ -0,0 +1,108 @@
+<?php
+/**
+ * Class File | src/File.php
+ *
+ * A single File with his available actions
```

Comment:
"his" should be "it's" since a file is not human and can't have a gender. 😄 This also occurs in other files.